src/backend/resourcemanager/resqueuemanager.c

```diff
@@ -2216,6 +2223,9 @@ int acquireResourceFromResQueMgr(ConnectionTrack  conntrack,
 	int						res			= FUNC_RETURN_OK;
 	DynResourceQueueTrack	queuetrack	= conntrack->QueueTrack;
 
+	elog(LOG, "ConnID %d. Expect query resource for session "INT64_FORMAT,
```

Comment:
Two values needs to be provided here for "%d" and "INT64_FORMAT". But here is only "conntrack->SessionID"